translations/fr.json

```diff
@@ -699,22 +671,21 @@
   "gamestatus.destroyed": "détruite(s)",
   "gamestatus.tasks": "Tâches {0}/{1}",
   "gamestatus.tasks.complete": "Complétée!",
-
   "settings.title": "PARAMÈTRES",
   "settings.volume": "Volume sonore {0}",
   "settings.soundeffects": "Effets sonores",
   "settings.ambience": "Ambiance",
-  "settings.tribemusic": "Son de tribu",
-  "settings.suggestions": "Suggestions de jeu",
+  "settings.tribemusic": "Musique de tribu",
+  "settings.suggestions": "Suggestions",
   "settings.info": "Information sur les constructions",
   "settings.confirm": "Confirmer la fin du tour",
-  "settings.saveexit": "SAUVEGARDER \n\u0026\n QUITTER",
+  "settings.saveexit": "SAUVEGARDER \u0026 QUITTER",
```

Comment:
I have no idea to resolve this button.. I try to put the & at the next line but it's not beautiful... Maybe the big button is a better choice. What do you think about it?